config/clickhouse/dist_config.xml

```diff
@@ -0,0 +1,15 @@
+<yandex>
```

Comment:
Let's call this file `remote_servers.xml` since that's what it contains (in line with our other config file names)

Reply:
Ok!